.github/workflows/update_protos.yml

```diff
@@ -3,7 +3,7 @@ name: Update Protos + Open PR
 on:
   push:
     branches:
-      - "RSDK-2091-auto-PR-api-changes"
+      - 'RSDK-2091-auto-PR-api-changes'
```

Comment:
we can delete this entire `push: ...` section actually - it was left over from testing

Reply:
boo, GH isn't letting me select all 3 lines to delete as a suggestion :(